modules/network-observability-architecture.adoc

```diff
@@ -0,0 +1,13 @@
+//Module included in the following assemblies:
+//
+// network_observability/understanding-network-observability.adoc
+
+:_content-type: CONCEPT
+[id="network-observability-architecture_{context}"]
+= Network Observablity Operator architecture
+image::network-observability-architecture.png[Network Observability eBPF export architecture]
+The Network Observability Operator provides the `FlowCollector` API, which is instantiated at installation and configured to reconcile the `flowlog-pipeline`, the eBPF agent, and the console-plugin components. Only a single `FlowCollector` per cluster is supported.
```

Comment:
```suggestion
The Network Observability Operator provides the `FlowCollector` API, which is instantiated at installation and configured to reconcile the `eBPF agent`, the `flowlog-pipeline` and the `netobserv-plugin` components. Only a single `FlowCollector` per cluster is supported.
```

Maybe keep the schema naming + ordering ?

Reply:
Good point. This will definitely make connecting the diagram and the text more straightforward.